optimize/backend/src/main/java/io/camunda/optimize/service/db/repository/es/TaskRepositoryES.java

```diff
@@ -37,9 +39,12 @@ public class TaskRepositoryES extends TaskRepository {
 
   private static final Logger LOG = org.slf4j.LoggerFactory.getLogger(TaskRepositoryES.class);
   private final OptimizeElasticsearchClient esClient;
+  private final ElasticSearchConfiguration configuration;
 
-  public TaskRepositoryES(final OptimizeElasticsearchClient esClient) {
+  public TaskRepositoryES(
+      final OptimizeElasticsearchClient esClient, final ConfigurationService configurationService) {
     this.esClient = esClient;
+    this.configuration = configurationService.getElasticSearchConfiguration();
```

Comment:
rather than a parameter, could we not just declare `ConfigurationService` directly? It is also a bean 🤷

Reply:
fixed